src/Dev/Tasks/i18nTextCollectorTask.php

```diff
@@ -55,9 +55,10 @@ protected function execute(InputInterface $input, PolyOutput $output): int
      */
     protected function getIsMerge(InputInterface $input): bool
     {
-        $merge = $input->getOption('merge');
-        // merge=0 or merge=false will disable merge
-        return !in_array($merge, ['0', 'false']);
+        if (true === $input->hasParameterOption(['--no-merge'], true)) {
```

Comment:
This doesn't seem right, it won't work in HTTP

If I simply put in 

`var_dump($input->getParameterOption(['--merge']));`

And then navigate using my browser (not CLI):

```
/dev/tasks/i18nTextCollectorTask?locale=en&merge=0 => bool(false) 
/dev/tasks/i18nTextCollectorTask?locale=en&merge=1 => bool(true) 
/dev/tasks/i18nTextCollectorTask?locale=en&merge=false => bool(false) 
/dev/tasks/i18nTextCollectorTask?locale=en&merge=true => bool(true) 
```

So we'd need to update this logic to respect &merge=<something> AND the converse of &no-merge=<something>

Reply:
For HTTP requests, `?merge=0` should work the same way that `--no-merge` works in CLI. So checking `$input->getOption('merge')` is the correct way to check it.